README.md

```diff
@@ -17,6 +15,52 @@ This will launch your browser, where the demo page will be displayed (at http://
 
```

Comment:
The existing instructions:
```This will launch a server, with a link to the demo page (at http://localhost:3000).```

...are incorrect on my computer.  The actual message returned on my machine is:
```Project is running at http://127.0.0.1:8080/```

We should do one of the following:

1) Weaken these instructions, changing them to: "This will start a server. Open a browser and navigate to the localhost URL mentioned in the console output."
2) Add "demo page opens on start" to this pull request

I'm leaning towards option 1, with a separate ticket to be opened for "Open demo page on start".  What do you think?